docs/backend-api/resources/tracking/tracker/alarm_mode.md

```diff
@@ -15,28 +15,46 @@ Gets the state of alarm mode of device.
 | :------ | :------ | :----- | :----- |
 | tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked | int | 999199 |
 
-#### example
+#### examples
+
+=== "HTTP POST application/json
 
 ```abap
 $ curl -X POST '{{ extra.api_example_url }}/tracker/alarm_mode/read' \
--H 'Content-Type: application/json' \ 
--d '{"tracker_id": "999199", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
+    -H 'Content-Type: application/json' \ 
+    -d '{"tracker_id": "123456", "hash": "a6aa75587e5c59c32d347da438505fc3"}'
+```
+
+=== "HTTP POST application/x-www-form-urlencoded"
+
+```abap
+$ curl -X POST '{{ extra.api_example_url }}tracker/alarm_mode/read' \
+    -d 'tracker_id=123456' \
+    -d 'hash=a6aa75587e5c59c32d347da438505fc3' \
+```
+
+=== "For Postman"
+
+```abap
+{{ extra.api_example_url }}/tracker/alarm_mode/read?tracker_id=123456&hash=a6aa75587e5c59c32d347da438505fc3
 ```
 
 #### response
 
 ```json
 {
  "success": true,
- "enabled": {boolean} // true if alarm mode is enabled
+ "enabled": true
 }
 ```
 
+* enabled - true if alarm mode enabled
```

Comment:
Missed a dot at the end. `enabled` should be escaped (as all fields and entity names).